### ISSUE TITLE
Version 5.1.1 RubyGems release?

### ISSUE THREAD

**Issue by BenEinwechter** [rails-sqlserver/activerecord-sqlserver-adapter#602]
Any chance you'll be releasing version `5.1.1` to RubyGems soon? Or, until then, at least adding the `v5.1.1` tag to this repo?

My project requires `v5.1.1` but since there's no GitHub tag yet, I've had to link directly to a SHA in my Gemfile, which isn't ideal.

```
gem 'activerecord-sqlserver-adapter', git: 'https://github.com/rails-sqlserver/activerecord-sqlserver-adapter', ref: '4a4ce1e37e7ef4ea6d388342e9bfbbc22c6fe734'
```

Thanks so much for this great gem!

**Comment by metaskills** [comment]
Why would we need 5.1.1? You might be thinking the adapter mirrors EXPLICIT major/minor/tiny versions of Rails and that is not true. See here: https://github.com/rails-sqlserver/activerecord-sqlserver-adapter#about-the-adapter

You should do this and let us take care of things using semver.

```ruby
gem 'activerecord-sqlserver-adapter', 
         git: 'https://github.com/rails-sqlserver/activerecord-sqlserver-adapter'
```

**Comment by BenEinwechter** [comment]
Thanks for the quick reply! 

Yeah, I was not trying to match the Rails tiny/patch version. Instead, I was upgrading to Rails 5.1 and seeing an error when using the `5.1.0` version of `activerecord-sqlserver-adapter` currently on https://rubygems.org. I saw `5.1.1` was available according to this project's CHANGELOG.md, so when I forced the gem to 5.1.1 by using the `git:` reference in my Gemfile, the error went away.

Just now though I downgraded to 5.1.0, and I'm not seeing the error I was getting before, and I don't remember what the error was, so maybe I don't actually need 5.1.1 after all.

For now, I'll either just use 5.1.0 or link directly to this git repo.

Feel free to close this issue. Thanks!
